add --target flag examples (#8367)
fishtown-analytics/docs.getdbt.com: website/docs/reference/global-configs/about-global-configs.md

## Available flags
Because the values of `flags` can differ across invocations, we strongly advise against using `flags` as an input to configurations or dependencies (`ref` + `source`) that dbt resolves [during parsing](/reference/parsing#known-limitations).

### Common flag examples

> The `--target` flag is commonly used to specify which target (environment) to use when running dbt commands. For example:

Author: ```suggestion
Use the `--target` flag to specify which target (environment) to use when running dbt commands. For example:
```

```bash
dbt run --target dev
dbt run --target prod
dbt build --target staging
```

> This allows you to run the same dbt project against different environments without modifying your configuration files. The target must be defined in your `profiles.yml` file. Learn more about [connection profiles and targets](/docs/core/connect-data-platform/connection-profiles).

Reviewer: Suggest to link directly to the section since the page is a bit long
```suggestion
The `--target` flag allows you to run the same dbt project against different environments without modifying your configuration files. Define the target in your `profiles.yml` file. Learn more about [connection profiles and targets](/docs/core/connect-data-platform/connection-profiles#understanding-targets-in-profiles).
```
